### ISSUE TITLE
Find inexpensive LED par38

### ISSUE THREAD

**Issue by burlexpo** [burlexpo/gbe10#7]
We want to replace the lamps in the R40 borderlights with LED par38.  We need 36 (48?) of them, plus a few spares.  The cheapest source Scratch has been able to find is about $15 each.

Seth says there's a place in Maine where we can get them for around $5 each.

**Comment by SRWILHELM** [comment]
my source can get them at $9.00 each 
3,000k color temp, dimmable.

**Comment by burlexpo** [comment]
Sounds good to me.  @BenignCremator do we need 36 or 48? And, I'll buy a few extra for replacements.

Seth, whats the best way to do this? You buy them and I repay you and get them from you? Or can I order them directly and get them shipped to me? Or even drive up and get them.

**Comment by eclecticmagpie** [comment]
What wattage, candlepower, beam angle?  Just wondering.

Mike Bergman
www.prospecthillforge.com
www.vintagedancers.org/teadance
www.isebastiani.com
reply-to: eclectic@mit.edu

**Comment by SRWILHELM** [comment]
i am getting the rest of the stats on them. it will prolly be me buy them and you send me a check so it  "looks" like we are using them in Maine. then i will figure a way to get them down to you. 

-Seth

**Comment by SRWILHELM** [comment]
11 watt dimmable
3000K
30 degree flood
800 lumens
25,000 hours

**Comment by BenignCremator** [comment]
On 03/11/2015 07:43 AM, SRWILHELM wrote:

> 11 watt dimmable
> 3000K
> 30 degree flood
> 800 lumens
> 25,000 hours

800 Lumens seems dim, they sound like 75W equivs.  We are currently 
using 250W bulbs at 3600 Lumens. and some 150W that I can not find specs 
on.  Do they have anything brighter?

Hunter

**Comment by usigel4ss** [comment]
At the moment I'm not aware of anything on the commercial market for the average consumer that goes above a 74w equivalent. You'd need to look into specialty suppliers like those who sell the bulbs for retrofitting stadiums or street lights, but I'm not sure you'll find the socket type you need. Poke around at the Phillips site and maybe they have a way to ask that sort of a question. Or you may just need to cave into a florescent bulb at the right lumen and temperature.

I can't track that down at the moment as I'm busy with an illustrator project that I need to have done by tomorrow.

Lia

**Comment by burlexpo** [comment]
There are a ton of 150W equivalents available, and even some 250W equivalents.  The problem is most of them are around $50 per bulb.  I've never seen a fluorescent come close to 3K lumens.  

These are 150W flood equivalents at 3K lumens -- and there are brighter ones on the site.
https://www.1000bulbs.com/category/led-par38-150w-equal-fl-3000k/ but they're $42 each.

There's a 2200 LM for about $25 at https://www.1000bulbs.com/product/116085/PL-PAR38W1502200302D.html

**Comment by eclecticmagpie** [comment]
What's the problem that we're trying to solve?

**Comment by burlexpo** [comment]
The lights we used to illuminate the cyc draw too much power. We need bright, at least 150 W equivalent, LED par 38 bulbs that aren't $40 each because at that price we could just replace the border lights with six 3-foot long LED wall washers.

**Comment by usigel4ss** [comment]
You have to be careful with ColorBlazes. Once you use them you really never want to go back to standard gels unless you deal with video. (those things look really wonky on camera due to LED technology and camera light receptors still not quite friendly)

**Comment by BenignCremator** [comment]
On 03/11/2015 07:55 PM, usigel4ss wrote:

> You have to be careful with ColorBlazes. Once you use them you really
> never want to go back to standard gels unless you deal with video.
> (those things look really wonky on camera due to LED technology and
> camera light receptors still not quite friendly)

Yeah, but that depends a lot of what is being lit as well.  This is for 
the cyc lights, so the light is falling on a neutral color fabric, not 
the dancers, clothes, etc.  Also, we are looking at broad spectrum LED 
bulbs, and using traditional filters on them.  That should mitigate the 
problem to some degree.

Hunter

**Comment by usigel4ss** [comment]
Yes. Arisia has rented them(colorblazes) many times in the past as cyc uplights. They typically rent them as part of the gear order from Jim Housell in NJ, but I'm not sure if some place more local would have them at reasonable price. _shrug_

Going back to the individual bulb route, it wasn't to long ago a 75w equivalent bulb was $20 or more a pop. It takes a few years of production for the brighter bulbs to come down in price.

**Comment by burlexpo** [comment]
I can get these for $17 each in quantity; that's the best I've found.  They're 120W equivalents; 1800 lumens in 120° Flood (1900 in 30° spot), 5000 K temperature.  

http://shop.ledwholesalers.com/index.php?route=product/product&path=36_61&product_id=551&sort=p.price&order=DESC

**Comment by eclecticmagpie** [comment]
Let's not forget that the prices of LED bulbs is still dropping fairly
rapidly.  We should probably revisit the question in November or December;
our options may be different.

Mike

**Comment by SRWILHELM** [comment]
i can have my electrician keep his eyes open and let me know if something pops up. He can get rebate style pricing through Efficiency Maine on LEDs and such

**Comment by SRWILHELM** [comment]
higher lumen are prohibitively expensive.

**Comment by burlexpo** [comment]
OK, as @eclecticmagpie says, we can table this and revisit in October/November.  We have to order lamps by November 15, 2015.

**Comment by eclecticmagpie** [comment]
One last thing you an do now is decide what the go/no-go price is, for both lamps for the old fixture and for wall washers (remembering that the latter _could_ drop in price faster than the individual lamps do), and if you and Hunter consider cheap CFLs an alternative, what _their_ price needs to be relative to the other two choices.

If those decisions are made now, while you (Scratch) are already thinking about this, then in October, Seth and/or Hunter can just go ahead and look at the current market and pick something without you getting involved again.

Just my two cents.  I have no horse in this race, as long as whatever you get doesn't make buzzing noises or generate heaps of EMI.
